src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2.AdvisorScenarioTest.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Test;
+using Xunit;
+
+namespace Sql2.Tests.ScenarioTests
+{
+    /// <summary>
+    /// Contains tests for Server, Elastic Pool and Database Advisors
+    /// </summary>
+    public class Sql2AdvisorScenarioTests : TestBase
+    {
+        private const string ResourceGroupName = "WIRunnersProd";
```

Comment:
I don't understand what needs to be setup here. If you meant setting up the resource group mentioned above, we have mocked responses for each API call in tests so there is no need to do so.